editor/editor.tsx

```diff
@@ -866,26 +851,34 @@ function Editor(props: RouteComponentProps) {
                         </span>
                         <span
                             title={
-                                code.length <= LIMIT_CLIPBOARD_LEN
+                                stringifySpec(goslingSpec || '').length <= LIMIT_CLIPBOARD_LEN
                                     ? `Copy unique URL of current view to clipboard (limit: ${LIMIT_CLIPBOARD_LEN} characters)`
                                     : `The current code contains characters more than ${LIMIT_CLIPBOARD_LEN}`
                             }
                             className={
-                                code.length <= LIMIT_CLIPBOARD_LEN
+                                stringifySpec(goslingSpec || '').length <= LIMIT_CLIPBOARD_LEN
                                     ? 'side-panel-button'
                                     : 'side-panel-button side-panel-button-not-active'
                             }
                             onClick={() => {
-                                if (code.length <= LIMIT_CLIPBOARD_LEN) {
+                                if (stringifySpec(goslingSpec || '').length <= LIMIT_CLIPBOARD_LEN) {
                                     // copy the unique url to clipboard using `<input/>`
-                                    const crushedSpec = encodeURIComponent(JSONCrush.crush(code));
+                                    const crushedSpec = encodeURIComponent(
+                                        JSONCrush.crush(stringifySpec(goslingSpec || ''))
+                                    );
                                     const url = `${window.location.origin}${window.location.pathname}?full=${isHideCode}&spec=${crushedSpec}`;
-                                    const element = document.getElementById('spec-url-exporter');
-                                    (element as any).type = 'text';
-                                    (element as any).value = url;
-                                    (element as any).select();
-                                    document.execCommand('copy');
-                                    (element as any).type = 'hidden';
+
+                                    // fix execCommand("copy") is obsolete
```

Comment:
```suggestion
```

Reply:
sorry for overlooking this issue while merging the master branch.
It seems to be related to the URL cache.
Changing the URL prefix to "https://gosling.js.org/" (as in the commit https://github.com/gosling-lang/gosling.js/pull/701/commits/bdf12cfce11fa9fcd731f91b4437a27c59bb552f) will fix this issue.